docs/stdlib/datetime.rst

```diff
@@ -369,29 +369,28 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         ...        <cal::relative_duration>'1 hour';
         {<cal::local_time>'23:00:00'}
 
-    If an arithmetic operation results in a day that doesn't exist in the given
-    month, the last day of the month is used instead.
+    If an arithmetic operation results in a day that is nonexistent in the
+    given month, the last day of the month will be used instead:
 
     .. code-block:: edgeql-repl
 
       db> select <cal::local_datetime>"2021-01-31T15:00:00" +
       ...        <cal::relative_duration>"1 month";
       {<cal::local_datetime>'2021-02-28T15:00:00'}
 
-
-    During arithmetic operations involving a ``relative_duration`` consisting
-    of multiple components (units), higher-order components are applied first,
-    followed by lower-order elements.
+    For arithmetic operations using :eql:type:`cal::relative_duration`
+    consisting of multiple components (units), higher-order components are
+    applied first, followed then by lower-order elements.
 
     .. code-block:: edgeql-repl
 
       db> select <cal::local_datetime>"2021-04-30T15:00:00" +
       ...        <cal::relative_duration>"1 month 1 day";
       {<cal::local_datetime>'2021-05-31T15:00:00'}
 
-    Compare this to adding up the same duration components separately
-    with higher-order units first and then lower-order, which produces
-    the same result as above:
+    This behavior is different from adding up the same components separately
```

Comment:
```suggestion
    If you add the same components split into separate durations, adding the
```

The behavior here is the same if you add the split components from higher-order to lower-order. Also, reworded a bit to try to improve clarity.